### PR TITLE
CM-526: accessibility fixes

### PR DESCRIPTION
**Pull request by foley-lynn**
### Jira Ticket:
CM-526
CM-527
CM-528

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/CM-526
https://bcparksdigital.atlassian.net/browse/CM-527
https://bcparksdigital.atlassian.net/browse/CM-528

### Description:
Combined a few minor accessibility fixes.

### CODE REVIEW

**Review by ayumi-oxd — CHANGES_REQUESTED** [review]
One nitpicking, but everything looks great!
I was able to:
- check that the home page has a heading
- check that there's no empty Read More/Less button
- skip to content with tab controll

File: src/staging/src/components/search/mainSearch.js [hunk 1]
```diff
@@ -19,9 +19,7 @@ const MainSearch = () => {
 
   return (
     <div className="parks-search-wrapper">
-      <div className="parks-search-header">
-        Find a park
-      </div>
+      <h1 class="text-white">Find a park</h1>
```
[the remark this answers is not in the document]
Comment (foley-lynn) [reply src/staging/src/components/search/mainSearch.js]: Good catch, thanks.

**Review by ayumi-oxd — APPROVED** [review]

File: src/staging/src/components/header.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
+import SkipToContent from "./skipToContent"
 import BetaBanner from "../components/betaBanner"
 import MegaMenu from "./megaMenu.js"
 
 export default function Header({ content = [] }) {
 
   return (
     <>
+      <SkipToContent></SkipToContent>
```
Comment (nataliasabadysh) [inline src/staging/src/components/header.js:11]: same as <SkipToContent></SkipToContent>

we can do in React  <SkipToContent /> 
same thing, just for info

**Review by nataliasabadysh — APPROVED** [review]

File: src/staging/src/components/header.js (hunk 2 above)
Comment (foley-lynn) [reply src/staging/src/components/header.js:11]: Hah, that's what I thought, but I figured I'd be consistent with the next line `<BetaBanner></BetaBanner>`.

I'll merge this, but feel free to change later.

**Pull request merged**